src/Components/BehaviorsModal/Internal/Widgets/DataHistoryWidget/DataHistoryWidget.tsx

```diff
@@ -51,6 +52,7 @@ import {
     IDataHistoryWidgetStyleProps,
     IDataHistoryWidgetStyles
 } from './DataHistoryWidget.types';
+import isChromatic from 'chromatic/isChromatic';
```

Comment:
@darsney Is this safe for consumers that don't build Storybook?

Reply:
Yeah @darsney , I initially committed changes to call isChromatic from the preview.js and if it is I was wrapping it with a StableDate context to pass along the static date to the components using a hook as we do for stable guids. but if it will cause a problem in build I need to go back to the changes that I reverted.

Reply:
I reverted back to the changes on this commit https://github.com/microsoft/iot-cardboard-js/commit/3b5e509e634c9d1923879ab5132b40cd44bf1918#diff-98b614e1838b171ee71c04450a8f1a562753193fb7d4f53a4de8b9b5a7e980ee I made earlier to use context. Added context and called isChromatic at the very top most level of Storybook in preview.js checking if it is Chromatic environment wrapping the story with stable date. And using a hook (which internally calls useContext) in the components I am checking if that stable date exist, if exist use it, if not use Date.now as the end date to show in the Highcharts.